src/routers/peers.py

```
@@ -331,10 +333,23 @@ async def get_working_representation(
 async def search_peer(
     workspace_id: str = Path(..., description="ID of the workspace"),
     peer_id: str = Path(..., description="ID of the peer"),
-    query: str = Body(..., description="Search query"),
+    search: schemas.MessageSearchOptions = Body(
+        ..., description="Message search parameters "
+    ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)